### PR TITLE
Hackathon - Barcelona 2023 Groups

### PR DESCRIPTION
**Pull request by ewels**
Created new component for hackathon groups to cut down on the HTML markup that needs to be written for these pages.

~~WIP - can't get it to even build currently. Need to go and sleep, hoping that someone else can continue this effort 🙏🏻~~
_[Edit]: Got it to work. Now I'm tired._

Outreach group meeting notes with group info [here](https://hackmd.io/Y3SeU7IDSaekc4rdd0B1Vg).

### CODE REVIEW

File: src/content/events/2023/hackathon-october-2023.mdx [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+- Evenings: in-person social events in Barcelona, as part of the Nextflow Summit program.
+- In person + online quiz! Test your nf-core knowledge!
+- Scavenger Hunt
+  - Take photos of you completing as many challenges as you can and share them on the channel!
+- Hackathon bingo
+  - Grab your [bingo card](https://nfcore-bingo.web.app/?game=nf-core-hackathon) and join in the fun.
+- Hidden socks in Gather
+  - Take screenshots of as many as you can find!
+  - See the sock hunt information board in Gather for more information.
```
Comment (christopher-hakkaart) [inline src/content/events/2023/hackathon-october-2023.mdx:193]:
```suggestion

**In person:**

- Evening social events in Barcelona, as part of the Nextflow Summit program.

**Hybrid (in-person and online)**

- nf-core quiz! Test your nf-core knowledge!
- Scavenger Hunt
  - Take photos of you completing as many challenges as you can and share them on the channel!
- Hackathon bingo
  - Grab your [bingo card](https://nfcore-bingo.web.app/?game=nf-core-hackathon) and join in the fun.
- Hidden socks in Gather
  - Take screenshots of as many as you can find!
  - See the sock hunt information board in Gather for more information.
```

File: src/content/events/2023/hackathon-october-2023.mdx [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+Please see the [Summit website Hackathon Agenda](https://summit.nextflow.io/barcelona/agenda/hackathon/#events) for details.
+Note that there is a [calendar view](https://summit.nextflow.io/barcelona/agenda/calendar/)
+with links that you can use to subscribe / add to your personal calendar.
+
+## Venue: Hotel SB Glow
+
+The in-person hackathon will be taking place at Hotel SB Glow.
+More information about this venue can be found the the [Nextflow Summit website](https://summit.nextflow.io/).
+
+[👉🏻 Google Maps 🗺️](https://goo.gl/maps/6uKCKkbciLAR5qgSA)
```
Comment (christopher-hakkaart) [inline src/content/events/2023/hackathon-october-2023.mdx:61]:
```suggestion
The in-person hackathon will be taking place at [Hotel SB Glow](https://goo.gl/maps/6uKCKkbciLAR5qgSA).
More information about this venue can be found the the [Nextflow Summit website](https://summit.nextflow.io/).
```

**Review by christopher-hakkaart — APPROVED** [review]
Two small suggestions otherwise 👍

**Comment by ewels** [comment]
Sorry, I polluted this PR with a change in the linting config. Apologies for all the whitespace diff.

File: src/content/events/2023/hackathon-october-2023.mdx [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+    - Adding new functionality to existing pipelines
+    - Updating pipelines to use the latest nf-core template, via template syncs
+    - Teams formed around work on specific pipelines
+    - Azure megatests roll-out in pipelines
+
+</HackathonGroup>
+
+<HackathonGroup title="Modules & Subworkflows" color="#1f6feb" proj_url="https://github.com/orgs/nf-core/projects/47/views/5" leaders={{
+    'JoseEspinosa': 'Jose Espinosa-Carrasco',
+    'ggabernet': 'Gisela Gabernet'
```
Comment (FriederikeHanssen) [inline src/content/events/2023/hackathon-october-2023.mdx:142]:
```suggestion
    'FriederikeHanssen': 'Friederike Hanssen'
```

File: src/content/events/2023/hackathon-october-2023.mdx [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+
+## Project groups
+
+<HackathonGroup title="Pipelines" color="#bf4b8a" proj_url="https://github.com/orgs/nf-core/projects/47/views/4" leaders={{
+    'drpatelh': 'Harshil Patel',
+    'adamrtalbot': 'Adam Talbot',
+    'MaxUlysse': 'Maxime Garcia',
+    'Emiller88': 'Edmund Miller',
+    'sateeshperi': 'Sateesh Peri',
+}}>
```
Comment (FriederikeHanssen) [inline src/content/events/2023/hackathon-october-2023.mdx:122]:
```suggestion
<HackathonGroup title="Pipelines" color="#bf4b8a" proj_url="https://github.com/orgs/nf-core/projects/47/views/4" leaders={{
    'drpatelh': 'Harshil Patel',
    'adamrtalbot': 'Adam Talbot',
    'MaxUlysse': 'Maxime Garcia',
    'Emiller88': 'Edmund Miller',
    'sateeshperi': 'Sateesh Peri',
}} slack=https://nfcore.slack.com/archives/C05RYK1F4PP>
```

File: src/content/events/2023/hackathon-october-2023.mdx [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+    - Updating pipelines to use the latest nf-core template, via template syncs
+    - Teams formed around work on specific pipelines
+    - Azure megatests roll-out in pipelines
+
+</HackathonGroup>
+
+<HackathonGroup title="Modules & Subworkflows" color="#1f6feb" proj_url="https://github.com/orgs/nf-core/projects/47/views/5" leaders={{
+    'JoseEspinosa': 'Jose Espinosa-Carrasco',
+    'ggabernet': 'Gisela Gabernet'
+}}>
```
Comment (FriederikeHanssen) [inline src/content/events/2023/hackathon-october-2023.mdx:143]:
```suggestion
}} slack=https://nfcore.slack.com/archives/C05RYK38561>
```

File: src/content/events/2023/hackathon-october-2023.mdx [hunk 6]
```diff
@@ -0,0 +1,240 @@
...
+    - Creating new modules, see issue list
+    - Stubs for all modules
+
+</HackathonGroup>
+
+<HackathonGroup title="Infrastructure" color="#9e6a03" proj_url="https://github.com/orgs/nf-core/projects/47/views/3" leaders={{
+    'mashehu': 'Matthias Hörtenhuber',
+    'mirpedrol': 'Júlia Mir Pedrol',
+    'nvnieuwk': 'Nicolas Vannieuwkerke'
+}}>
```
Comment (FriederikeHanssen) [inline src/content/events/2023/hackathon-october-2023.mdx:159]:
```suggestion
}} slack=https://nfcore.slack.com/archives/C05RYK591QD>
```

File: src/content/events/2023/hackathon-october-2023.mdx [hunk 7]
```diff
@@ -0,0 +1,240 @@
...
+    - New lint tests (especially for modules, eg. stubs)
+    - Google megatests
+    - nf-validation plugin
+
+</HackathonGroup>
+
+<HackathonGroup title="Nextflow backbone" color="#238636" proj_url="https://github.com/orgs/nf-core/projects/47/views/2" leaders={{
+    'bentsherman': 'Ben Sherman',
+    'jordeu': 'Jordi Deu-Pons'
+}}>
```
Comment (FriederikeHanssen) [inline src/content/events/2023/hackathon-october-2023.mdx:179]:
```suggestion
}} slack=https://nfcore.slack.com/archives/C05RM1ASWBZ>
```

File: src/content/events/2023/hackathon-october-2023.mdx [hunk 8]
```diff
@@ -0,0 +1,240 @@
...
+:::
+
+### In person:
+
+-   Evening social events in Barcelona, as part of the Nextflow Summit program.
+
+### Hybrid (in-person and online)
+
+-   nf-core quiz! Test your nf-core knowledge!
+-   Scavenger Hunt
```
Comment (FriederikeHanssen) [inline src/content/events/2023/hackathon-october-2023.mdx:208]:
```suggestion
-   Scavenger Hunt: https://nfcore.slack.com/archives/C05RYKSDL93
```

File: src/content/events/2023/hackathon-october-2023.mdx (hunk 4 above)
Comment (ewels) [reply src/content/events/2023/hackathon-october-2023.mdx:122]: Sorry, should have said - `slack` here is literally the Slack channel name, not the URL. Also needs to be in quotes.

**Pull request merged**